src/box/iproto.cc

```diff
@@ -3167,6 +3293,18 @@ iproto_do_cfg_f(struct cbus_call_msg *m)
 		case IPROTO_CFG_STAT:
 			iproto_fill_stat(iproto_thread, cfg_msg);
 			break;
+		case IPROTO_CFG_OVERRIDE: {
+			uint64_t high_bit_mask = UINT64_C(1) << 63;
+			bool set = (cfg_msg->rq_type & high_bit_mask) != 0;
+			uint32_t rq_type = cfg_msg->rq_type;
+			if (set)
+				mh_i32_put(iproto_thread->request_handlers,
+					   &rq_type, NULL, NULL);
+			else
+				mh_i32_remove(iproto_thread->request_handlers,
+					      &rq_type, NULL);
```

Comment:
Nit: It'd be nice to have some assertions here ensuring that `mh_i32_put` doesn't replace anything and `mp_i32_remove` actually deletes something.

Reply:
Fixed: added assertions. Also made sure we send this IPROTO configuration message only when necessary.